src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,180 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private int size;
+    private Node head;
+    private Node tail;
+
     @Override
     public void add(T value) {
+        if (size < 2) {
+            addHead(value);
+            return;
+        }
+
+        Node node = new Node<>(tail, value, null);
+        tail.next = node;
+        tail = node;
+        size++;
     }
 
     @Override
     public void add(T value, int index) {
+        if (index < 0) {
```

Comment:
try to make it like this
![image](https://github.com/mate-academy/jv-linked-list/assets/20326858/dec2b623-efb4-49bc-8663-8de44b7a7ec8)